fw/rbcx-coprocessor/include/utils/TickTimer.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "FreeRTOS.h"
+#include "task.h"
+
+class TickTimer {
```

Comment:
By se mohly hodit ty FreeRTOS timery https://www.freertos.org/RTOS-software-timer.html

Reply:
To dělá něco jiného - pustí to tvůj callback na tasku zvlášť, což mi přijde jako hrozný overkill, když já prostě potřebuju změřit (poměrně nepřesně) čas uvnitř té naší hlavní smyčky.